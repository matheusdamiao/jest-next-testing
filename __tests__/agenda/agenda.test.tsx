import Agenda from "@/pages/agenda";
import { render, screen } from "@testing-library/react";

describe("Agenda", () => {
  it("renders an agenda title", () => {
    const { container } = render(<Agenda />);
    expect(container).toBeInTheDocument();
    expect(screen.getByTestId("title")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
