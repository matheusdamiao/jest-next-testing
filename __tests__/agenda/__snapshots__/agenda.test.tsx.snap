// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agenda renders an agenda title 1`] = `
<div>
  <div>
    <h3
      data-testid="title"
    >
      Escolha o melhor dia e horário para você
    </h3>
    <div
      class="calendly-inline-widget calendario"
      data-url="https://calendly.com/matheusdamiao/pre-briefing?hide_event_type_details=1&hide_gdpr_banner=1"
    />
  </div>
</div>
`;
